src/operations/contract.lisp

```diff
@@ -275,7 +275,9 @@ If we have a non-null peduncle edge (F -> C above), then we need to tell its sou
                      :petal-children petal-child-edges
                      :dryad dryad-address)
               (set-up-result fresh-blossom)
-            (values)))))))
+            (sync-rpc (make-message-add-macrovertex :address fresh-blossom)
```

Comment:
is async ok?

Reply:
probably but i think im okay with the speed-synchronicity tradeoff here